ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -0,0 +1,184 @@
+ * of choice (although only credit cards will be allowed
+ * initially).
+ * Keep in mind that the rest of the "add credit card" process
+ * is handled @https://www.pivotaltracker.com/story/show/157838293
+ */
+import * as React from "react";
+import I18n from "../../i18n";
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  Grid,
+  H1,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Row,
+  Text,
+  View
+} from "native-base";
+import { Image, StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import AppHeader from "../../components/ui/AppHeader";
+import Modal from "../../components/ui/Modal";
+import variables from "../../theme/variables";
+
+// Images
+const bankLogo = require("../../../img/wallet/payment-methods/bank.png");
+const creditCardLogo = require("../../../img/wallet/payment-methods/creditcard.png");
+const mobileLogo = require("../../../img/wallet/payment-methods/mobile.png");
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type State = Readonly<{
+  isTosModalVisible: boolean;
+}>;
+
+const paymentMethods: ReadonlyArray<any> = [
+  {
+    navigateTo: "", // TODO: add route when destination is available @https://www.pivotaltracker.com/story/show/157588719
+    name: I18n.t("wallet.methods.card.name"),
+    maxFee: I18n.t("wallet.methods.card.maxFee"),
+    icon: creditCardLogo
+  },
+  {
+    navigateTo: "",
+    name: I18n.t("wallet.methods.bank.name"),
+    maxFee: I18n.t("wallet.methods.bank.maxFee"),
+    icon: bankLogo
+  },
+  {
+    navigateTo: "",
+    name: I18n.t("wallet.methods.mobile.name"),
+    maxFee: I18n.t("wallet.methods.mobile.maxFee"),
+    icon: mobileLogo
+  }
+];
+
+const AddMethodStyle = StyleSheet.create({
+  paymentMethodEntry: {
+    marginLeft: 0,
+    paddingRight: 0,
+    height: 75
+  },
+  transactionText: {
+    fontSize: variables.fontSize1,
+    color: "#a6a6a6" // WIP to be updated to variables.white.darken(.35)
+  },
+  centeredContents: {
+    alignItems: "center"
+  },
+  containedImage: {
+    width: "100%",
+    resizeMode: "contain"
+  }
+});
+
+export class AddPaymentMethodScreen extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isTosModalVisible: false
+    };
+  }
+
+  public render(): React.ReactNode {
+    const { navigate } = this.props.navigation;
+
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button
+              transparent={true}
+              onPress={_ => this.props.navigation.goBack()}
+            >
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.addPaymentMethodTitle")}</Text>
+          </Body>
+        </AppHeader>
+        <Content>
+          <Text>{I18n.t("wallet.chooseMethod")}</Text>
+          <View spacer={true} large={true} />
+          <List
```

Comment:
see [this comment](https://github.com/teamdigitale/italia-app/pull/159#pullrequestreview-122583149) by @fpersico

Reply:
Out of curiosity, does either of you know whether Native Base's `ListItem` component will be maintained? I could find [this](https://github.com/GeekyAnts/NativeBase/issues/759#issuecomment-342725652), which I interpreted as saying that it will still be available, but I may be missing something you know

Reply:
List will be deprecated, ListItem can be used inside FlatList. CHeck this sample

https://github.com/GeekyAnts/native-base-example-flatlist-stickyHeaders/blob/master/src/StickyList.js